docs/source/releases/1.16.3/clean-up-terminal-output-for-call-cmd-and-testing.yaml

```diff
@@ -0,0 +1,115 @@
+testing and installation:
+- title: 'Update call_cmd to explicitly request to use_print or use_logging, rather than toggle with use_logging'
```

Comment:
Call_cmd is currently exclusively used in the pytests, it was created to help better control where output was going for pytest based integration tests